### DISCUSSION TITLE
Dose Kor support open source model

### DISCUSSION THREAD

**Discussion by gouse95** [eyurtsev/kor#265] [Q&A] [answered]
If it supports open source LLM's please provide list of models

**Comment by eyurtsev** [comment, marked as answer]
It uses a prompt based technique rather than function calling or JSON mode, so it basically works with any language or chat model. Look at langchain documentation to identify which models you might be able to use.

Quality of results depends on the prompt and on the LLMs used (see guidelines in documentation)
